CASC/PortfolioMode.cpp

```diff
@@ -146,6 +155,7 @@ bool PortfolioMode::searchForProof()
   return performStrategy(property);
 }
 
+// TODO this name confuses me
```

Comment:
Cue bikeshedding about what we should call it. I find the original confusing because it suggests that it does something like `runSlice`, but actually it generates a schedule and runs it.